### PR TITLE
* lib/rubygems.rb, lib/rubygems/*, test/rubygems/*: Update rubygems-2.6.1

### PR DESCRIPTION
**Pull request by segiddins**
\c @indirect @hsbt

### CODE REVIEW

**Comment by hsbt** [comment]
@segiddins Thank you for your preparation. I will merge this into Ruby trunk.

File: test/rubygems/test_gem_commands_build_command.rb [hunk 1]
```diff
@@ -62,7 +62,7 @@ def test_execute_missing_file
     assert_equal '', @ui.output
     assert_equal "ERROR:  Gemspec file not found: some_gem\n", @ui.error
   end
-
+  
   def test_can_find_gemspecs_without_dot_gemspec
```
Comment (nobu) [inline test/rubygems/test_gem_commands_build_command.rb]: Seems unnecessary

Comment (segiddins) [inline test/rubygems/test_gem_commands_build_command.rb]: This is pulling in directly from rubygems

**Comment by segiddins** [comment]
Updated to now be 2.6.1

**Pull request closed without merging**